packages/base/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2019 Ericsson and others
```

Comment:
Nitpicking: shouldn't be 2019-2021?

Reply:
It was copied from the root directory. I'll update all the Lisense files.